AdtSampleApp/SampleClientApp/CommandLoop.cs

```diff
@@ -54,10 +53,10 @@ public async Task CommandCreateModels(string[] cmd)
                     r.Close();
                     dtdlList.Add(dtdl);
                 }
-                Response<ModelData[]> res = await client.CreateModelsAsync(dtdlList);
+                Response<DigitalTwinsModelData[]> res = await client.CreateModelsAsync(dtdlList);
                 Log.Ok($"Model(s) created successfully!");
-                foreach (ModelData md in res.Value)
-                    LogResponse(md.Model);
+                foreach (DigitalTwinsModelData md in res.Value)
+                    LogResponse(md.DtdlModel);
             }
             catch (RequestFailedException e)
```

Comment:
Add a catch specifically for conflict case?
// If a model already exists
(RequestFailedException e) when (e.Status == (int)HttpStatusCode.Conflict)

Reply:
I'm not updating their sample to be better just to use the new SDK.